common-money.lic

```diff
@@ -33,16 +33,18 @@ module DRCM
     amount
   end
 
-  def wealth
-    fput 'wealth'
-    waitfor('Wealth:')
-    wealth = matchfind '\\(? copper Kronars\\)\\.', 'No Kronars?'
-    wealth == '.' ? 0 : wealth.to_i
+  def wealth(hometown)
+    case hometown
+    when "Riverhaven"
+      wealth = DRC.bput("wealth lirum", '\(\d+ copper Lirums\)', "No Lirums").scan(/\d+/).first.to_i
```

Comment:
You've got a map between hometown and its currency. That data is already in `base-town` (hometown['exchange']['currency']).

I think we should move currency up a level so it is a property of the town itself, but that's not necessary in this PR.